runtime/centrifuge/src/lib.rs

```diff
@@ -806,22 +806,18 @@ impl pallet_nft::Config for Runtime {
 
 parameter_types! {
 	pub const BridgePalletId: PalletId = common_types::ids::BRIDGE_PALLET_ID;
-	pub HashId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &sp_io::hashing::blake2_128(&common_types::ids::CHAIN_BRIDGE_HASH_ID));
 	pub NativeTokenId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &sp_io::hashing::blake2_128(&common_types::ids::CHAIN_BRIDGE_NATIVE_TOKEN_ID));
-	pub const NativeTokenTransferFee: u128 = NATIVE_TOKEN_TRANSFER_FEE;
-	pub const NftTransferFee: u128 = NFT_TOKEN_TRANSFER_FEE;
+	pub const NativeTokenTransferFeeKey: FeeKey = FeeKey::NativeTokenTransfer;
```

Comment:
**Careful!**: The new balance is 1 * CFG (The default value for the storage of `pallet-keys`), and the old value is `NATIVE_TOKEN_TRANSFER_FEE` which is 2000 * CFG. Is this a problem? Should we use some kind of runtime upgrade?

Reply:
I think as soon as it is enacted we can trigger a council motion to correct it. I dont think is a huge issue. Alternatively, what we can do is to split this PR in 2, one to add the Key enum, then we populate the new value and two to add the code changes.

Reply:
It seems interesting the idea 👍🏻. Tomorrow we can talk deeply about it.